MetaMask/metamask-extension
# Feat/add collectible manually

Explanation:  
- Hooks up CollectiblesController to the NFT (manually) add collectible view.
- Adds success and failure toasts for the manual add action.

Manual testing steps:  
  - Navigate to the NFTs tab and click on "Add NFT"
  - Add an NFT (contract address/token ID) that you actually own (if none you can get some [here](https://faucet.paradigm.xyz/) for testnets)
  - Check that there is a dismissable "Collectible was successfully added!" toast
  - Redo this process with an NFT that you don't own
  - Check for a dismissable failure toast "Collectible was not added because: This collectible is not owned by the user"
  
  TODO:
- [ ] add unit & e2e tests?
